### PR TITLE
Variables Hint + Guard + Verbose Error Mode

### PR DESCRIPTION
**Pull request by JakeCooper**
Not pictured: running commands with -v will now show panicErr and stacktrace if it occurs

image

### CODE REVIEW

File: cmd/variables.go [hunk 1]
```diff
@@ -49,6 +51,9 @@ func (h *Handler) VariablesSet(ctx context.Context, req *entity.CommandRequest)
 	updatedEnvNames := make([]string, 0)
 	for _, kvPair := range req.Args {
 		parts := strings.SplitN(kvPair, "=", 2)
+		if len(parts) != 2 {
```
Comment (gschier) [inline cmd/variables.go:54]: Nice

File: controller/panic.go [hunk 2]
```diff
@@ -10,6 +10,7 @@ import (
 )
 
 func (c *Controller) SendPanic(ctx context.Context, panicErr string, stacktrace string, command string) (bool, error) {
+	fmt.Println(panicErr, stacktrace)
```
Comment (gschier) [inline controller/panic.go:13]: Did you mean to leave this here?

File: main.go [hunk 3]
```diff
@@ -117,21 +117,24 @@ func init() {
 		RunE:    contextualize(handler.Variables, handler.Panic),
 	})
 	variablesCmd.AddCommand(&cobra.Command{
-		Use:   "get",
-		Short: "Get the value of a variable",
-		RunE:  contextualize(handler.VariablesGet, handler.Panic),
-		Args:  cobra.MinimumNArgs(1),
+		Use:     "get key",
```
Comment (gschier) [inline main.go:120]: What the heck? I always thought this was just the command name to use for Cobra. I didn't know you could put help strings afterwards

**Review by gschier — APPROVED** [review]

File: controller/panic.go (hunk 2 above)
Comment (JakeCooper) [reply controller/panic.go:13]: I did not

File: main.go (hunk 3 above)
Comment (JakeCooper) [reply main.go:120]: Ya me neither lol. Guess we know now!

image

**Pull request merged**